sql/handler.cc

```diff
@@ -7312,18 +7312,17 @@ bool Vers_parse_info::check_sys_fields(const Lex_table_name &table_name,
   return true;
 }
 
-static bool check_period_field(const Create_field* f, const char* name,
-                               const char* period_name)
+bool Table_period_info::check_field(const Create_field* f, const Lex_ident& f_name) const
```

Comment:
You see, this check is only applied to application time period, and does not to SYSTEM_TIME.
Still SYSTEM_TIME is also described with Table_period_info